jaeger-thrift/build.gradle

```diff
@@ -43,6 +48,11 @@ shadowJar {
     classifier 'thrift92'
 }
 
+shadowJar {
```

Comment:
Don't we just want the single shaded jar, with the dependencies relocated?

Reply:
Should I just remove the one from jaeger-core and have then `jaeger-thrift-okhttp381`?

http://central.maven.org/maven2/io/jaegertracing/jaeger-core/0.28.0/

Reply:
Sorry, I think I get a better understanding now. Did you mean to have the shadowed okhttp on the main artifact as well? Or did you mean to have the optional okhttp shadow artifact for thrift, instead of for jaeger-core?

Reply:
I just changed this PR to include both shadowed thrift and okhttp on the same "shadow" artifact